### PR TITLE
Updates Error Handling to Prevent Application Crashes

### LINKED ISSUE #15: Web frontend going blank and unresponsive after event without a body
When submitting an event without a JSON body/payload the web UI goes blank and stops responding:

image

After reloading the page in the browser the UI starts working again.

### PR DESCRIPTION
**Pull request by JeffNeff**
Small update to the handling of the `data` property when rendering the incoming events. When an event with a malformed/missing data property comes in you will now get an alert message telling you to check the logs where the full event can be found.

This closes [#15](https://github.com/n3wscott/sockeye/issues/15) in the `sockeye` repo

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -55,11 +55,20 @@ export class App extends Component {
...
-    this.setState( {
+    if (data["data"] != null){
+      al.push(data);
+      this.setState( {
         events: al
     });
+    return;
+    } 
+    if (data["data"] === undefined){
+      alert("Event received with an invalid or missing data payload. Check the console for more information");
```
Comment (n3wscott) [inline src/App.js]: maybe not an alert :D

Maybe we can use the alert component instead? https://material-ui.com/components/alert/

Comment (JeffNeff) [reply src/App.js]: yes of course!

**Comment by JeffNeff** [comment]
Screen Shot 2021-09-09 at 10 03 13 AM
How does this look?

**Review by n3wscott — APPROVED** [review]
Awesome, thanks!!

LGTM

**Pull request merged**